borgmatic/config/schema.yaml

```diff
@@ -1602,6 +1602,88 @@ properties:
                 example:
                     - start
                     - finish
+    zabbix:
+        type: object
+        additionalProperties: false
+        properties:
+            itemid:
+                type: integer
+                description: |
+                    The itemid to publish to
+                    for details.
```

Comment:
Sorry, I am terrible at writing descriptions. In Zabbix, each host that you are monitoring has items associated with them. The items store values. For example, it might store text, integer etc.

Itemid is the ID that uniquely identifies the item and is unique across all hosts and items. This is why the host and key do not need to be specified here.

Basically, this is the id of the item that will hold the data.

Reply:
Got it. It sounds a little bit like a key in a traditional key/value store. And this looks like the description of the concept? https://www.zabbix.com/documentation/current/en/manual/config/items

If so, maybe just say something like: "The ID of the Zabbix item used for collecting data."

Reply:
that is way better. Agreed: https://github.com/borgmatic-collective/borgmatic/pull/85/commits/7aff22536df75998f174b83aa16066e87465f66c